.github/workflows/prod_deploy.yaml

```diff
@@ -7,10 +7,17 @@ jobs:
   invoke-prod-deploy:
     runs-on: ubuntu-latest
     steps:
-      - name: "deploy"
+      - name: deploy
         run: |
-          curl --fail --silent --show-error -X POST \
+          echo DEPLOY_BATCH_URL=$(curl --fail --silent --show-error -X POST \
```

Comment:
This looks strange to me, echo and assign in the same line of bash? Maybe my bash isn't quite up to date. I'll try this when I'm near a computer next 😀

Reply:
I'm not sure either that this will work, but it's GitHub's somewhat strange way to set environment variables:
```bash
echo KEY=val >> $GITHUB_ENV
```
This will make the `$KEY` environment variable available to subsequent steps in the workflow.